docs/docs/notification-center/iframe-embed.md

```diff
@@ -5,33 +5,49 @@ You can find the embed code in the `Settings` page within the Admin Panel. It wi
 
 ```html
 <script>
-  (function(n,o,t,i,f) {
-    n[i] = {}; var m = ['init']; n[i]._c = [];m.forEach(me => n[i][me] = function() {n[i]._c.push([me, arguments])});
-    var elt = o.createElement(f); elt.type = "text/javascript"; elt.async = true; elt.src = t;
-    var before = o.getElementsByTagName(f)[0]; before.parentNode.insertBefore(elt, before);
+  (function (n, o, t, i, f) {
+    n[i] = {};
+    var m = ['init'];
+    n[i]._c = [];
+    m.forEach(
+      (me) =>
+        (n[i][me] = function () {
+          n[i]._c.push([me, arguments]);
+        })
+    );
+    var elt = o.createElement(f);
+    elt.type = 'text/javascript';
+    elt.async = true;
+    elt.src = t;
+    var before = o.getElementsByTagName(f)[0];
+    before.parentNode.insertBefore(elt, before);
   })(window, document, 'https://embed.novu.co/embed.umd.min.js', 'novu', 'script');
```

Comment:
Could we skip the prettier here? It's just easier to copy as a "minified" and more compact version of this one here.